mobile/src/context/BookContext.tsx

```diff
@@ -47,12 +53,38 @@ export const BookProvider: React.FC = ({ children }) => {
     });
   }
 
+  function fetchPopularBooks(): void {
+    dispatch({ type: 'API_CALL_STARTED' });
```

Comment:
this should be split out to a separate action as well, since API_CALL_STARTED affects the loading bool for getBooks()